src/input/InputSDL.cpp

```diff
@@ -117,10 +139,10 @@ void InputSDLWorker::run()
 
         case SDL_JOYAXISMOTION:
         {
-          int deadband = 32768 *0.05;
+          int deadband = 32768 * 0.15;
```

Comment:
5% causes the xbox joystick to trigger events with the slightest movement. Since some joysticks also act as a button when you press them, you need a bit more dead space to prevent doing the wrong thing.

15% was a guess, and for my slightly older controller its right at the point where the stick starts to put up some resistance. 5% was in the dead space where you can wobble the stick.